docs/release/release_0_4_19.md

```diff
@@ -0,0 +1,350 @@
+please make an issue on the napari repo requesting the feature to be exposed!
+
+napari is now pydantic 2 compatible. At this moment we are using `pydantic.v1` to achieve this. 
+Additionally, for this release, the napari conda bundle will be shipped with pydantic v1, but
+we expect that in the next release we will ship the bundle with pydantic v2.
+Please, if you are a plugin developer and your plugin uses pydantic, ensure
+that it is compatible with pydantic v2 ([napari/napari/#6358](https://github.com/napari/napari/pull/6358)).
+
+We have fixed problems with rendering some labels in incorrect color by sequence of Pull Requests:
+[napari/napari/#3308](https://github.com/napari/napari/pull/3308),
+[napari/napari/#6411](https://github.com/napari/napari/pull/6411),
+[napari/napari/#6467](https://github.com/napari/napari/pull/6467),
+[napari/napari/#6439](https://github.com/napari/napari/pull/6439).
+[napari/napari/#6607](https://github.com/napari/napari/pull/6607).
+This also improves performance of rendering labels.
+However, there are still some limitations.
+Best performance is for labels encoded as (u)int8.
+As GPU API accept 8 and 16 bits int textures and float textures,
+for labels stored as (u)int8 and (u)int16 we could transfer data to GPU without preprocessing.
+
+For bigger data types we need to convert data, so it leads to bigger memory usage.
+Also best performance for bigger data requires to install numba library.
+
+So if you wold like for best user performance, please try to use the minimal possible data type.
+Even if, in the future, we'll use more advanced techniques for rendering labels on GPU, that will allow
+transferring any data without preprocessing, the GPU often offers less VRAM that computer has RAM.
+Also we experiment a little with current options and increase the size of
+send data (uint8, uint16 and float32) decrease performance of rendering on GPU.
+The test was performed on NVIDIA GeForce RTX 3060 with 12GB VRAM.
+
+If you have any questions or suggestions regarding napari core, for example, 
+on how to adapt your plugin to be compatible with future napari versions, please get in touch!
+We have weekly community meetings, for which you can find the schedule [here](https://napari.org/stable/community/meeting_schedule.html).
+We would like to hear from you.
+
+- Use a shader for low discrepancy label conversion ([napari/napari/#3308](https://github.com/napari/napari/pull/3308))
+- Automatic recognition of hex colour strings in layer data ([napari/napari/#6102](https://github.com/napari/napari/pull/6102))
+- Postpone qt_viewer deprecation to 0.6.0 ([napari/napari/#6283](https://github.com/napari/napari/pull/6283))
+- Pydantic 2 compatibility using `pydantic.v1`  ([napari/napari/#6358](https://github.com/napari/napari/pull/6358))
+- Initial deprecations for Labels API ([napari/napari/#6542](https://github.com/napari/napari/pull/6542))
```

Comment:
Are these meant to be removed and/or moved to their corresponding paragraphs?

Reply:
Currently, they are automatically added by script. I'm not sure how to handle them properly. Maybe needs to be removed before merging.